messaging/client.go

```diff
@@ -0,0 +1,110 @@
+package messaging
+
+import (
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/Shopify/sarama"
+	"github.com/google/uuid"
+	"github.com/nivista/steady/timer"
+	"github.com/nivista/steady/utils"
+)
+
+// Client is a synchronous messaging client to the queue.
+type Client struct {
+	producer   sarama.SyncProducer
+	partitions int
+	topic      string
+}
+
+// NewClient returns a new Client.
+func NewClient() *Client {
+	client := Client{}
+
+	client.topic = os.Getenv("TIMER_TOPIC")
+
+	partitions, err := strconv.Atoi(os.Getenv("TIMER_PARTITIONS"))
+	if err != nil {
+		panic(err)
```

Comment:
why do we panic here but return an error for database client?

Reply:
Ya that is a good point. I'll return error in both places.